client/question/components/LearnerViewDirective.js

```diff
@@ -1310,6 +1310,17 @@ tie.directive('learnerView', [function() {
             $scope.privacyModalIsDisplayed = true;
           }
         }
+
+        /**
+         * Refreshes UI if the window loads at 0 height or width (IFrame Firefox
+           Case)
+         */
+        var millisecondDelay = 500;
```

Comment:
I think it's fine to inline this, since it's not reused anywhere and the name doesn't have an intrinsic meaning. Or alternatively rename it to something like initialIntervalDelayMsecs.

Reply:
Done. Removed the named variable as it wasn't necessary with $timeout.